src/main/scala/elevate/rise/rules/traversal.scala

```diff
@@ -111,6 +111,16 @@ object traversal {
     override def toString = s"body($s)"
   }
 
+  case class lambdaBodyWithName(
+    s: Identifier => Strategy[Rise]
+  ) extends Strategy[Rise] {
```

Comment:
I am not sure if we want to keep this strategy. I do not think that I have any remaining uses currently.